tests/boards/native_posix/cpu_wait/src/main.c

```diff
@@ -211,7 +211,7 @@ static void np_timer_isr_test_replacement(const void *arg)
  * The kernel is configured as NOT-tickless, and the default tick period is
  * 10ms
  */
-static void test_cpu_hold_with_interrupts(void)
+ZTEST(native_cpu_hold, test_cpu_hold_with_interrupts)
 {
 #if defined(CONFIG_BOARD_NATIVE_POSIX)
```

Comment:
This test is a no-op if `!CONFIG_BOARD_NATIVE_POSIX`. Can we move the `ifdef` up so it's just never declared in that case?

Reply:
Done.